internal/pkg/vault/management.go

```diff
@@ -247,3 +247,275 @@ func (c *Client) ConfigureConsulAccess(secretStoreToken string, bootstrapACLToke
 	})
 	return err
 }
+
+func (c *Client) CreateOrUpdateIdentity(secretStoreToken string, name string, metadata map[string]string, policies []string) (string, error) {
+	urlPath := path.Join(namedEntityAPI, name)
+	request := CreateUpdateEntityRequest{Metadata: metadata, Policies: policies}
+	response := CreateUpdateEntityResponse{}
+
+	_, err := c.doRequest(RequestArgs{
+		AuthToken:            secretStoreToken,
+		Method:               http.MethodPost,
+		Path:                 urlPath,
+		JSONObject:           request,
+		BodyReader:           nil,
+		OperationDescription: "Create/Update Entity",
+		ExpectedStatusCode:   http.StatusOK,
```

Comment:
fyi- usually the http response for create or update is 201 NoContent but i'm ok with custom one like here

Reply:
It does return a response.
https://developer.hashicorp.com/vault/api-docs/secret/identity/entity#create-update-entity-by-name